openwisp_monitoring/monitoring/migrations/0011_remove_metric_description.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.0.3 on 2020-05-30 11:28
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('monitoring', '0010_metric_configuration'),
+    ]
+
+    operations = [
+        migrations.RemoveField(model_name='metric', name='description',),
+    ]
```

Comment:
There's no need to have two migrations. I would merge both into just one

Reply:
Well, there are two separate issues here. I preferred a separate migration as it's easier when we migrate back for an case (just in case). Let me know if I should merge them both, I would prefer to keep it this way.

Reply:
I still don't see the benefit. Also we don't migrate back often

Reply:
Very well then, I shall merge both of them